cluster/gce/gci/configure_helper_test.go

```diff
@@ -148,7 +148,10 @@ func (c *ManifestTestCase) mustLoadPodFromManifest() {
 }
 
 func (c *ManifestTestCase) tearDown() {
-	os.RemoveAll(c.kubeHome)
+	err := os.RemoveAll(c.kubeHome)
+	if err != nil {
```

Comment:
optional:
`if err := os.RemoveAll(c.kubeHome); err != nil {`
...

Reply:
Thanks for you suggestion, I changed it.
Cloud you please add a lgtm label for this PR again?